src/db/influxdb/config.rs

```diff
@@ -79,12 +79,12 @@ pub enum AnalyticsChoice {
     BaseTokenActivity,
     BlockActivity,
     DailyActiveAddresses,
-    InputOutputDegrees,
     LedgerOutputs,
     LedgerSize,
     MilestoneSize,
     OutputActivity,
     ProtocolParameters,
+    TransactionDistribution,
```

Comment:
Your name doesn't provide any benefit for its length, and the word `degree` has no meaning in this context. I'd rather call it something like `TransactionSize`, which is really what we are measuring.

Reply:
`TransactionInputOutputCountDistribution` ... If you find a more generic term for `InputOutput` that is also shorter I take it :trollface: `Size` would have a completely different meaning that for `LedgerSize` analytic.

Reply:
It's simply too long and doesn't add anything. `Size` is perfectly adequate. Just because we're measuring two different kinds of size with two different metrics doesn't mean anything.

Reply:
Sorry. I think you're wrong about this. A transaction with more inputs/outputs than another is in no way bigger than another. And you're also wrong about `degree` not having a meaning in this context. That's the exact term used by Nikita in this context, so we should use the same terminology. My last offer is `TransactionDegreeDistribution`. It's not much longer than your proposal and it contains the term that researchers/mathematicians use in this context.